[2023-03-22T16:39:40.732-0400] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: my_api_connection.api_sensor scheduled__2023-03-21T00:00:00+00:00 [queued]>
[2023-03-22T16:39:40.737-0400] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: my_api_connection.api_sensor scheduled__2023-03-21T00:00:00+00:00 [queued]>
[2023-03-22T16:39:40.737-0400] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-22T16:39:40.737-0400] {taskinstance.py:1280} INFO - Starting attempt 2 of 1
[2023-03-22T16:39:40.737-0400] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-22T16:39:40.741-0400] {taskinstance.py:1300} INFO - Executing <Task(HttpSensor): api_sensor> on 2023-03-21 00:00:00+00:00
[2023-03-22T16:39:40.744-0400] {standard_task_runner.py:55} INFO - Started process 76286 to run task
[2023-03-22T16:39:40.747-0400] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'my_api_connection', 'api_sensor', 'scheduled__2023-03-21T00:00:00+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/cities.py', '--cfg-path', '/var/folders/71/zhs_yc7n05l7c97rnhn92bb80000gn/T/tmpyo32od95']
[2023-03-22T16:39:40.747-0400] {standard_task_runner.py:83} INFO - Job 59: Subtask api_sensor
[2023-03-22T16:39:40.780-0400] {task_command.py:388} INFO - Running <TaskInstance: my_api_connection.api_sensor scheduled__2023-03-21T00:00:00+00:00 [running]> on host sunandas-laptop.local
[2023-03-22T16:39:40.798-0400] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=my_api_connection
AIRFLOW_CTX_TASK_ID=api_sensor
AIRFLOW_CTX_EXECUTION_DATE=2023-03-21T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-21T00:00:00+00:00
[2023-03-22T16:39:40.799-0400] {http.py:122} INFO - Poking: weather?q=cities,states,US&units=standard&appid=94ca56afa3eee20f7100985fdfe682aa
[2023-03-22T16:39:40.801-0400] {base.py:73} INFO - Using connection ID 'my_api_connection' for task execution.
[2023-03-22T17:18:24.571-0400] {local_task_job.py:112} ERROR - Received SIGTERM. Terminating subprocesses
[2023-03-22T17:18:24.629-0400] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 76286. PIDs of all processes in the group: [76286]
[2023-03-22T17:18:24.630-0400] {process_utils.py:84} INFO - Sending the signal Signals.SIGTERM to group 76286
